go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/nephio-project/porch
 
 go 1.22
 
+replace github.com/GoogleContainerTools/kpt-functions-sdk/go/fn => github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220506190241-f85503febd54
```

Comment:
This is a pain. How do we handle this dependency going forward?

Reply:
Not really so much about the multi module situation but more about the sdk itself. It's pretty much dead right and we rely heavily on it

Reply:
I like the SDK, I think it is a good project, and I even have some contributions lined-up since Nephio R1. However I agree that it seems to be abandoned by Google, but nobody can take over, or even contribute who is not allowed to sign the Google CLA (like me). It should have been migrated to the kpt CNCF project by now, but that hasn't happened for more than a year. I asked about te migration here: https://github.com/kptdev/kpt/issues/4034, and also asked various Google people, but no news on the ETA.
Btw. the same goes to the kpt function catalog. It seems to be abandoned, and on the kpt slack there is at least one guy who also waits for the migration to be able to contribute. And an abandoned kpt function catalog doesn't help the adoption of neither kpt, nor Nephio.

Maybe we could fork those under Nephio? However I don't think that we have the capacity to actually maintain them.